realsense_camera/CMakeLists.txt

```diff
@@ -136,3 +136,9 @@ install(DIRECTORY rviz/
 install(FILES nodelet_plugins.xml
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
+
+include_directories(include ${catkin_INCLUDE_DIRS})
+
+add_executable(debug_tool src/debug_tool.cpp)
```

Comment:
Please move these up above the install section, above the test, with the commands for build the other binaries.